### ISSUE TITLE
Highlight which vertices were seeds in the graph UI

### ISSUE THREAD

**Issue by p013570** [gchq/gaffer-tools#742]

**Comment by p013570** [comment]
I've implemented a fairly naive solution, but it seems to work ok. It needs testing.

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
